apps/vr-tests-react-components/src/stories/Button/Button.stories.tsx

```diff
@@ -29,19 +29,19 @@ AsAnAnchor.storyName = 'As an anchor';
 
```

Comment:
## [🕵 FluentUIV0 [This is the changings] Open the Visual Regressions report to inspect the 1260 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval-app/index.html?organization=uifabric&project=fabricpublic&prId=27462&commitId=744f3f6f105fb04906305200cd1e60183fe4dd52&env=prod)

✅ undefined


      <details><summary><b>unknown</b> 1260 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Accordion.Accordion Default Bsize.chromium.png|0| Added|
|Accordion.Accordion Default Example.chromium.png|0| Added|
|Accordion.Accordion Example Custom Title.Focuses the accordion title.chromium.png|0| Added|
|Accordion.Accordion Example Custom Title.chromium.png|0| Added|
|Accordion.Accordion Example Rtl.chromium.png|0| Added|
|Accordion.Accordion Example.chromium.png|0| Added|
|Accordion.Accordion Exclusive Example.chromium.png|0| Added|
|Accordion.Accordion Exclusive Expanded Example.chromium.png|0| Added|
|Accordion.Accordion Panel Custom Content Example.chromium.png|0| Added|
|Accordion.Accordion Panel Custom Title Example.chromium.png|0| Added|
|Alert.Alert Default Bsize.chromium.png|0| Added|
|Alert.Alert Example Children Rtl.chromium.png|0| Added|
|Alert.Alert Example Danger Teams Dark.chromium.png|0| Added|
|Alert.Alert Example Danger Teams High Contrast.chromium.png|0| Added|
|Alert.Alert Example Danger Teams.chromium.png|0| Added|
|Alert.Alert Example Danger.chromium.png|0| Added|
|Alert.Alert Example Dismiss Action Rtl.chromium.png|0| Added|
|Alert.Alert Example Dismiss Action.chromium.png|0| Added|
|Alert.Alert Example Dismiss Actions.chromium.png|0| Added|
|Alert.Alert Example Dismissible Teams Dark.chromium.png|0| Added|
|Alert.Alert Example Dismissible Teams High Contrast.chromium.png|0| Added|
|Alert.Alert Example Dismissible Teams.chromium.png|0| Added|
|Alert.Alert Example Dismissible.chromium.png|0| Added|
|Alert.Alert Example Fitted.chromium.png|0| Added|
|Alert.Alert Example Header.chromium.png|0| Added|
|Alert.Alert Example Icon.chromium.png|0| Added|
|Alert.Alert Example Important Message.chromium.png|0| Added|
|Alert.Alert Example Info Teams Dark.chromium.png|0| Added|
|Alert.Alert Example Info Teams High Contrast.chromium.png|0| Added|
|Alert.Alert Example Info Teams.chromium.png|0| Added|
|Alert.Alert Example Info.chromium.png|0| Added|
|Alert.Alert Example Oof Teams Dark.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams Dark.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams Dark.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams Dark.chromium.png|0| Added|
|Alert.Alert Example Oof Teams High Contrast.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams High Contrast.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams High Contrast.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams High Contrast.chromium.png|0| Added|
|Alert.Alert Example Oof Teams.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Oof Teams.chromium.png|0| Added|
|Alert.Alert Example Oof.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Oof.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Oof.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Oof.chromium.png|0| Added|
|Alert.Alert Example Rtl.chromium.png|0| Added|
|Alert.Alert Example Shorthand.chromium.png|0| Added|
|Alert.Alert Example Success.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams Dark.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams Dark.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams Dark.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams Dark.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams High Contrast.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams High Contrast.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams High Contrast.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams High Contrast.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Urgent Teams.chromium.png|0| Added|
|Alert.Alert Example Urgent.Focuses the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent.Hovers the action button.chromium.png|0| Added|
|Alert.Alert Example Urgent.Hovers the focused action button.chromium.png|0| Added|
|Alert.Alert Example Urgent.chromium.png|0| Added|
|Alert.Alert Example Warning.chromium.png|0| Added|
|Alert.Alert Example Width.chromium.png|0| Added|
|Alert.Alert Example.chromium.png|0| Added|
|Animation.Animation Example Delay.chromium.png|0| Added|
|Animation.Animation Example Direction.chromium.png|0| Added|
|Animation.Animation Example Duration.chromium.png|0| Added|
|Animation.Animation Example Fill Mode.chromium.png|0| Added|
|Animation.Animation Example Iteration Count.chromium.png|0| Added|
|Animation.Animation Example Timing Function.chromium.png|0| Added|
|Animation.Animation Example Visible.chromium.png|0| Added|
|Animation.Animation Example.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams Dark.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand Teams.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Action Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams Dark.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand Teams.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand.Focuses action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand.Focuses root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand.Hovers action.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand.Hovers root.chromium.png|0| Added|
|Attachment.Attachment Actionable Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Default Bsize.chromium.png|0| Added|
|Attachment.Attachment Description Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Example Rtl.chromium.png|0| Added|
|Attachment.Attachment Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Header Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Icon Example Shorthand.chromium.png|0| Added|
|Attachment.Attachment Progress Example Shorthand.chromium.png|0| Added|
|Avatar.Avatar Default Bsize.chromium.png|0| Added|
|Avatar.Avatar Example Excluded Initials Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Get Initials Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Icon Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Image Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Label Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Name Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Rtl.chromium.png|0| Added|
|Avatar.Avatar Example Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Size Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Square.chromium.png|0| Added|
|Avatar.Avatar Example Status Customization Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Status Image Shorthand.chromium.png|0| Added|
|Avatar.Avatar Example Status Shorthand.chromium.png|0| Added|
|Avatar.Avatar Usage Example Shorthand.chromium.png|0| Added|
|Box.Box Default Bsize.chromium.png|0| Added|
|Box.Box Example.chromium.png|0| Added|
|Box.Box Shorthand Example.chromium.png|0| Added|
|Breadcrumb.Breadcrumb Example Icon Divider.chromium.png|0| Added|
|Breadcrumb.Breadcrumb Example Sizes.chromium.png|0| Added|
|Breadcrumb.Breadcrumb Example.chromium.png|0| Added|
|Button.Button Default Bsize.chromium.png|0| Added|
|Button.Button Example Circular.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams Dark.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams High Contrast.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon Teams.chromium.png|0| Added|
|Button.Button Example Content And Icon.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Content And Icon.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Content And Icon.chromium.png|0| Added|
|Button.Button Example Disabled Focusable.chromium.png|0| Added|
|Button.Button Example Disabled Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Disabled Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams Dark.chromium.png|0| Added|
|Button.Button Example Disabled Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Disabled Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams High Contrast.chromium.png|0| Added|
|Button.Button Example Disabled Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Disabled Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Disabled Teams.chromium.png|0| Added|
|Button.Button Example Disabled.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Disabled.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Disabled.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Disabled.chromium.png|0| Added|
|Button.Button Example Emphasis.chromium.png|0| Added|
|Button.Button Example Flat.chromium.png|0| Added|
|Button.Button Example Fluid.chromium.png|0| Added|
|Button.Button Example Icon Only.chromium.png|0| Added|
|Button.Button Example Inverted.chromium.png|0| Added|
|Button.Button Example Loading.chromium.png|0| Added|
|Button.Button Example Overflow.chromium.png|0| Added|
|Button.Button Example Rtl.chromium.png|0| Added|
|Button.Button Example Size Shorthand.chromium.png|0| Added|
|Button.Button Example Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Teams Dark.chromium.png|0| Added|
|Button.Button Example Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Teams High Contrast.chromium.png|0| Added|
|Button.Button Example Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Teams.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams Dark.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams High Contrast.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand Teams.chromium.png|0| Added|
|Button.Button Example Text Shorthand.Clicks the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example Text Shorthand.Hovers the first button.chromium.png|0| Added|
|Button.Button Example Text Shorthand.chromium.png|0| Added|
|Button.Button Example Text.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams Dark.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams High Contrast.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip Teams.chromium.png|0| Added|
|Button.Button Example With Tooltip.Clicks the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example With Tooltip.Hovers the first button.chromium.png|0| Added|
|Button.Button Example With Tooltip.chromium.png|0| Added|
|Button.Button Example.Clicks the first button.chromium.png|0| Added|
|Button.Button Example.Focuses on the second button.chromium.png|0| Added|
|Button.Button Example.Hovers the first button.chromium.png|0| Added|
|Button.Button Example.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams Dark.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand Teams.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Circular Example Shorthand.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams Dark.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams Dark.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams Dark.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams Dark.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams High Contrast.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams High Contrast.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams High Contrast.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand Teams.chromium.png|0| Added|
|Button.Button Group Example Shorthand.Clicks the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand.Focuses on the second button.chromium.png|0| Added|
|Button.Button Group Example Shorthand.Hovers the first button.chromium.png|0| Added|
|Button.Button Group Example Shorthand.chromium.png|0| Added|
|Button.Button Usage Example Shorthand.chromium.png|0| Added|
|Card.Card Example Body.chromium.png|0| Added|
|Card.Card Example Centered.chromium.png|0| Added|
|Card.Card Example Context Menu.chromium.png|0| Added|
|Card.Card Example Disabled.chromium.png|0| Added|
|Card.Card Example Elevated.chromium.png|0| Added|
|Card.Card Example Expandable.chromium.png|0| Added|
|Card.Card Example Focusable Grid.chromium.png|0| Added|
|Card.Card Example Focusable Teams Dark.chromium.png|0| Added|
|Card.Card Example Focusable Teams High Contrast.chromium.png|0| Added|
|Card.Card Example Focusable Teams.chromium.png|0| Added|
|Card.Card Example Focusable.chromium.png|0| Added|
|Card.Card Example Footer.chromium.png|0| Added|
|Card.Card Example Header.chromium.png|0| Added|
|Card.Card Example Horizontal.chromium.png|0| Added|
|Card.Card Example Inverted.chromium.png|0| Added|
|Card.Card Example Preview.chromium.png|0| Added|
|Card.Card Example Quiet.chromium.png|0| Added|
|Card.Card Example Selectable Grid.chromium.png|0| Added|
|Card.Card Example Selected.chromium.png|0| Added|
|Card.Card Example Simple.chromium.png|0| Added|
|Card.Card Example Size.chromium.png|0| Added|
|Card.Card Example With Preview Teams Dark.chromium.png|0| Added|
|Card.Card Example With Preview Teams High Contrast.chromium.png|0| Added|
|Card.Card Example With Preview Teams.chromium.png|0| Added|
|Card.Card Example With Preview.chromium.png|0| Added|
|Card.Image Example Fluent.chromium.png|0| Added|
|Carousel.Carousel Example.chromium.png|0| Added|
|Carousel.Carousel Slide Animation Example.chromium.png|0| Added|
|Chat.Chat Example Comfy Refresh.chromium.png|0| Added|
|Chat.Chat Example Compact Teams Dark.chromium.png|0| Added|
|Chat.Chat Example Compact Teams High Contrast.chromium.png|0| Added|
|Chat.Chat Example Compact Teams.chromium.png|0| Added|
|Chat.Chat Example Compact.chromium.png|0| Added|
|Chat.Chat Example Content Position.chromium.png|0| Added|
|Chat.Chat Example Details Teams Dark.chromium.png|0| Added|
|Chat.Chat Example Details Teams High Contrast.chromium.png|0| Added|
|Chat.Chat Example Details Teams.chromium.png|0| Added|
|Chat.Chat Example Details.chromium.png|0| Added|
|Chat.Chat Example Header Override.chromium.png|0| Added|
|Chat.Chat Example Header.chromium.png|0| Added|
|Chat.Chat Example In Scrollable Shorthand.chromium.png|0| Added|
|Chat.Chat Example Reaction Group.chromium.png|0| Added|
|Chat.Chat Example Read Status.chromium.png|0| Added|
|Chat.Chat Example Rtl.chromium.png|0| Added|
|Chat.Chat Example Teams Dark.chromium.png|0| Added|
|Chat.Chat Example Teams High Contrast.chromium.png|0| Added|
|Chat.Chat Example Teams.chromium.png|0| Added|
|Chat.Chat Example.chromium.png|0| Added|
|Chat.Chat Message Example Badge Teams Dark.chromium.png|0| Added|
|Chat.Chat Message Example Badge Teams High Contrast.chromium.png|0| Added|
|Chat.Chat Message Example Badge Teams.chromium.png|0| Added|
|Chat.Chat Message Example Badge.chromium.png|0| Added|
|Chat.Chat Message Example Styled.chromium.png|0| Added|
|Chat.Message Reactions With Popup Teams Dark.chromium.png|0| Added|
|Chat.Message Reactions With Popup Teams High Contrast.chromium.png|0| Added|
|Chat.Message Reactions With Popup Teams.chromium.png|0| Added|
|Chat.Message Reactions With Popup.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked Mixed Teams Dark.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked Mixed Teams High Contrast.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked Mixed Teams.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked Mixed.chromium.png|0| Added|
|Checkbox.Checkbox Example Checked.chromium.png|0| Added|
|Checkbox.Checkbox Example Disabled.chromium.png|0| Added|
|Checkbox.Checkbox Example Label Flex Column.chromium.png|0| Added|
|Checkbox.Checkbox Example Label.chromium.png|0| Added|
|Checkbox.Checkbox Example Rtl.chromium.png|0| Added|
|Checkbox.Checkbox Example Toggle.Checks checkbox.chromium.png|0| Added|
|Checkbox.Checkbox Example Toggle.Focuses checkbox.chromium.png|0| Added|
|Checkbox.Checkbox Example Toggle.chromium.png|0| Added|
|Checkbox.Checkbox Example.Checks checkbox.chromium.png|0| Added|
|Checkbox.Checkbox Example.Focuses checkbox.chromium.png|0| Added|
|Checkbox.Checkbox Example.chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams Dark.Does not show tooltip on not today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams Dark.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams Dark.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams High Contrast.Does not show tooltip on not today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams High Contrast.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams.Does not show tooltip on not today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example Teams.chromium.png|0| Added|
|Datepicker.Datepicker Cell Example.Does not show tooltip on not today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Cell Example.chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows cleared calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows cleared input..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams Dark.Shows selected date in input with clear possibility..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows cleared calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows cleared input..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams High Contrast.Shows selected date in input with clear possibility..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams.Shows cleared calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams.Shows cleared input..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams.Shows selected date in input with clear possibility..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable.Shows cleared calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable.Shows cleared input..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable.Shows selected date in input with clear possibility..chromium.png|0| Added|
|Datepicker.Datepicker Example Clearable.chromium.png|0| Added|
|Datepicker.Datepicker Example Disabled.chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams Dark.Shows localized calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams High Contrast.Shows localized calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams.Shows localized calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings.Shows localized calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Localization Strings.chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Disabled date is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Month icon is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams Dark.Shows min max dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Disabled date is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Month icon is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.Shows min max dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams.Disabled date is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams.Month icon is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams.Shows min max dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date.Disabled date is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date.Month icon is not clickable..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date.Shows min max dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Min Max Date.chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams Dark.Keeps calendar open..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams Dark.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams High Contrast.Keeps calendar open..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams High Contrast.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams.Keeps calendar open..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example No Input.Keeps calendar open..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example No Input.chromium.png|0| Added|
|Datepicker.Datepicker Example Open.chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams Dark.Input shows error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams Dark.Shows datepicker popup through button and error is shown..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams Dark.Shows selected date in input without error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams High Contrast.Input shows error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams High Contrast.Shows datepicker popup through button and error is shown..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams High Contrast.Shows selected date in input without error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams.Input shows error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams.Shows datepicker popup through button and error is shown..chromium.png|0| Added|
|Datepicker.Datepicker Example Required Teams.Shows selected date in input without error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required.Input shows error..chromium.png|0| Added|
|Datepicker.Datepicker Example Required.Shows datepicker popup through button and error is shown..chromium.png|0| Added|
|Datepicker.Datepicker Example Required.Shows selected date in input without error..chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams Dark.Shows disabled dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams High Contrast.Shows disabled dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams.Shows disabled dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates.Shows disabled dates..chromium.png|0| Added|
|Datepicker.Datepicker Example Restricted Dates.chromium.png|0| Added|
|Datepicker.Datepicker Example Selected Date.chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams Dark.Shows datepicker calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams High Contrast.Shows datepicker calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams.Shows datepicker calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button.Shows datepicker calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Button.chromium.png|0| Added|
|Datepicker.Datepicker Example Standalone Calendar Input.chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams Dark.Shows calendar with hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams Dark.Shows datepicker popup through button..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams Dark.Shows selected date in calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams Dark.Shows selected date in input..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows calendar with hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows datepicker popup through button..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows selected date in calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams High Contrast.Shows selected date in input..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams.Shows calendar with hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams.Shows datepicker popup through button..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams.Shows selected date in calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams.Shows selected date in input..chromium.png|0| Added|
|Datepicker.Datepicker Example Today Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Today.Shows calendar with hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Today.Shows datepicker popup through button..chromium.png|0| Added|
|Datepicker.Datepicker Example Today.Shows selected date in calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Today.Shows selected date in input..chromium.png|0| Added|
|Datepicker.Datepicker Example Today.chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams Dark.Calendar is opened with selected week highlighted..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams Dark.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams Dark.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams High Contrast.Calendar is opened with selected week highlighted..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams High Contrast.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams High Contrast.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams.Calendar is opened with selected week highlighted..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Week Teams.chromium.png|0| Added|
|Datepicker.Datepicker Example Week.Calendar is opened with selected week highlighted..chromium.png|0| Added|
|Datepicker.Datepicker Example Week.Calendar is opened with the entire week highlighted on hover..chromium.png|0| Added|
|Datepicker.Datepicker Example Week.Shows calendar..chromium.png|0| Added|
|Datepicker.Datepicker Example Week.chromium.png|0| Added|
|Datepicker.Datepicker Example.chromium.png|0| Added|
|Datepicker.Datepicker First Week Day Example.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams Dark.Shows datepicker popup.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams Dark.Shows selected date.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows datepicker popup.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams High Contrast.Shows selected date.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams.Shows datepicker popup.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams.Shows selected date.chromium.png|0| Added|
|Datepicker.Datepicker Format Example Teams.chromium.png|0| Added|
|Datepicker.Datepicker Format Example.Shows datepicker popup.chromium.png|0| Added|
|Datepicker.Datepicker Format Example.Shows selected date.chromium.png|0| Added|
|Datepicker.Datepicker Format Example.chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams Dark.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams Dark.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams Dark.chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams High Contrast.chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example Teams.chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example.Shows datepicker popup..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example.Shows tooltip on today's date..chromium.png|0| Added|
|Datepicker.Datepicker Header Cell Example.chromium.png|0| Added|
|Datepicker.Datepicker Header Example.chromium.png|0| Added|
|Datepicker.Datepicker No Input Placeholder Example.chromium.png|0| Added|
|Datepicker.Datepicker Parse Example.chromium.png|0| Added|
|Dialog.Dialog Example Backdrop Teams Dark.chromium.png|0| Added|
|Dialog.Dialog Example Backdrop Teams High Contrast.chromium.png|0| Added|
|Dialog.Dialog Example Backdrop Teams.chromium.png|0| Added|
|Dialog.Dialog Example Backdrop.chromium.png|0| Added|
|Dialog.Dialog Example Callbacks.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams Dark.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams Dark.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams High Contrast.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams High Contrast.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Content Teams.chromium.png|0| Added|
|Dialog.Dialog Example Content.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Content.chromium.png|0| Added|
|Dialog.Dialog Example Footer.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Footer.chromium.png|0| Added|
|Dialog.Dialog Example Full Width.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Full Width.chromium.png|0| Added|
|Dialog.Dialog Example Header Action Teams Dark.chromium.png|0| Added|
|Dialog.Dialog Example Header Action Teams High Contrast.chromium.png|0| Added|
|Dialog.Dialog Example Header Action Teams.chromium.png|0| Added|
|Dialog.Dialog Example Header Action.chromium.png|0| Added|
|Dialog.Dialog Example Rtl.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Rtl.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams Dark.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams Dark.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams High Contrast.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams High Contrast.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Scroll Teams.chromium.png|0| Added|
|Dialog.Dialog Example Scroll.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Scroll.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams Dark.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams Dark.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams High Contrast.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams High Contrast.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content Teams.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content.Clicks the trigger button.chromium.png|0| Added|
|Dialog.Dialog Example Zoom Content.chromium.png|0| Added|
|Dialog.Dialog Example.chromium.png|0| Added|
|Divider.Divider Example Color Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Color Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Color Teams.chromium.png|0| Added|
|Divider.Divider Example Color.chromium.png|0| Added|
|Divider.Divider Example Content Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Content Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Content Teams.chromium.png|0| Added|
|Divider.Divider Example Content.chromium.png|0| Added|
|Divider.Divider Example Fitted Shorthand Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Fitted Shorthand Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Fitted Shorthand Teams.chromium.png|0| Added|
|Divider.Divider Example Fitted Shorthand.chromium.png|0| Added|
|Divider.Divider Example Important Shorthand Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Important Shorthand Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Important Shorthand Teams.chromium.png|0| Added|
|Divider.Divider Example Important Shorthand.chromium.png|0| Added|
|Divider.Divider Example Important.chromium.png|0| Added|
|Divider.Divider Example Rtl And Ltr Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Rtl And Ltr Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Rtl And Ltr Teams.chromium.png|0| Added|
|Divider.Divider Example Rtl And Ltr.chromium.png|0| Added|
|Divider.Divider Example Rtl Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Rtl Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Rtl Teams.chromium.png|0| Added|
|Divider.Divider Example Rtl.chromium.png|0| Added|
|Divider.Divider Example Shorthand Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Shorthand Teams.chromium.png|0| Added|
|Divider.Divider Example Shorthand.chromium.png|0| Added|
|Divider.Divider Example Size Teams Dark.chromium.png|0| Added|
|Divider.Divider Example Size Teams High Contrast.chromium.png|0| Added|
|Divider.Divider Example Size Teams.chromium.png|0| Added|
|Divider.Divider Example Size.chromium.png|0| Added|
|Divider.Divider Vertical Example Shorthand.chromium.png|0| Added|
|Dropdown.Dropdown Clearable Example.Clears the value.chromium.png|0| Added|
|Dropdown.Dropdown Clearable Example.Selects an item.chromium.png|0| Added|
|Dropdown.Dropdown Clearable Example.chromium.png|0| Added|
|Dropdown.Dropdown Custom Clearable Example Teams Dark.chromium.png|0| Added|
|Dropdown.Dropdown Custom Clearable Example Teams High Contrast.chromium.png|0| Added|
|Dropdown.Dropdown Custom Clearable Example Teams.chromium.png|0| Added|
|Dropdown.Dropdown Custom Clearable Example.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams Dark.Mouse hover on input.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams Dark.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams High Contrast.Mouse hover on input.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams High Contrast.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams.Mouse hover on input.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled Teams.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled.Mouse hover on input.chromium.png|0| Added|
|Dropdown.Dropdown Example Disabled.chromium.png|0| Added|
|Dropdown.Dropdown Example Freeform.chromium.png|0| Added|
|Dropdown.Dropdown Example Header Message.Shows header message.chromium.png|0| Added|
|Dropdown.Dropdown Example Header Message.chromium.png|0| Added|
|Dropdown.Dropdown Example Inline.chromium.png|0| Added|
|Dropdown.Dropdown Example Inverted.chromium.png|0| Added|
|Dropdown.Dropdown Example Loading Teams Dark.chromium.png|0| Added|
|Dropdown.Dropdown Example Loading Teams High Contrast.chromium.png|0| Added|
|Dropdown.Dropdown Example Loading Teams.chromium.png|0| Added|
|Dropdown.Dropdown Example Loading.chromium.png|0| Added|
|Dropdown.Dropdown Example Multiple.Opened dropdown with no items selected.chromium.png|0| Added|
|Dropdown.Dropdown Example Multiple.Opened dropdown with two items selected.chromium.png|0| Added|
|Dropdown.Dropdown Example Multiple.chromium.png|0| Added|
|Dropdown.Dropdown Example Position.Opens a dropdown.chromium.png|0| Added|
|Dropdown.Dropdown Example Position.chromium.png|0| Added|
|Dropdown.Dropdown Example Render.chromium.png|0| Added|
|Dropdown.Dropdown Example Rtl.RTL- Selects an item.chromium.png|0| Added|
|Dropdown.Dropdown Example Rtl.RTL- Shows list.chromium.png|0| Added|
|Dropdown.Dropdown Example Rtl.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Fluid.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple French Language.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Image And Content.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams Dark.Hovers first selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams Dark.Selects last selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams Dark.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.Hovers first selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.Selects last selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams High Contrast.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams.Hovers first selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams.Selects last selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple Teams.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple.Hovers first selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple.Selects last selected element.chromium.png|0| Added|
|Dropdown.Dropdown Example Search Multiple.chromium.png|0| Added|
|Dropdown.Dropdown Example Search.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams Dark.Focuses first item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams Dark.Focuses second item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams Dark.Focuses trigger.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams Dark.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams High Contrast.Focuses first item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams High Contrast.Focuses second item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams High Contrast.Focuses trigger.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams High Contrast.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams.Focuses first item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams.Focuses second item.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams.Focuses trigger.chromium.png|0| Added|
|Dropdown.Dropdown Example Teams.chromium.png|0| Added|
|Dropdown.Dropdown Example.Focuses first item.chromium.png|0| Added|
|Dropdown.Dropdown Example.Focuses second item.chromium.png|0| Added|
|Dropdown.Dropdown Example.Focuses trigger.chromium.png|0| Added|
|Dropdown.Dropdown Example.chromium.png|0| Added|
|Embed.Embed Example Video.chromium.png|0| Added|
|Embed.Embed Example You Tube.chromium.png|0| Added|
|Flex.Flex Example Columns.chromium.png|0| Added|
|Flex.Flex Example Input.chromium.png|0| Added|
|Flex.Flex Example Items Alignment.chromium.png|0| Added|
|Flex.Flex Example Media Card.chromium.png|0| Added|
|Flex.Flex Example Mixed Alignment.chromium.png|0| Added|
|Flex.Flex Example Nav Menu.chromium.png|0| Added|
|Form.Form Example Checkbox.chromium.png|0| Added|
|Form.Form Example Components.chromium.png|0| Added|
|Form.Form Example Datepicker.chromium.png|0| Added|
|Form.Form Example Error And Satisfactory.chromium.png|0| Added|
|Form.Form Example Rtl.chromium.png|0| Added|
|Form.Form Example Slider.chromium.png|0| Added|
|Form.Form Example.chromium.png|0| Added|
|Grid.Grid Example.chromium.png|0| Added|
|Header.Header Example Align.chromium.png|0| Added|
|Header.Header Example Color.chromium.png|0| Added|
|Header.Header Example Description Customization Shorthand.chromium.png|0| Added|
|Header.Header Example Description Customization.chromium.png|0| Added|
|Header.Header Example Description Shorthand.chromium.png|0| Added|
|Header.Header Example Description.chromium.png|0| Added|
|Header.Header Example Rtl.chromium.png|0| Added|
|Header.Header Example.chromium.png|0| Added|
|Header.Headers Example Shorthand.chromium.png|0| Added|
|Image.Image Example Avatar.chromium.png|0| Added|
|Image.Image Example Circular.chromium.png|0| Added|
|Image.Image Example Fluent.chromium.png|0| Added|
|Image.Image Example Rtl.chromium.png|0| Added|
|Image.Image Example.chromium.png|0| Added|
|Input.Input Example Clearable Teams Dark.Can be clearable.chromium.png|0| Added|
|Input.Input Example Clearable Teams Dark.chromium.png|0| Added|
|Input.Input Example Clearable Teams High Contrast.Can be clearable.chromium.png|0| Added|
|Input.Input Example Clearable Teams High Contrast.chromium.png|0| Added|
|Input.Input Example Clearable Teams.Can be clearable.chromium.png|0| Added|
|Input.Input Example Clearable Teams.chromium.png|0| Added|
|Input.Input Example Clearable.Can be clearable.chromium.png|0| Added|
|Input.Input Example Clearable.chromium.png|0| Added|
|Input.Input Example Disabled.chromium.png|0| Added|
|Input.Input Example Fluid.chromium.png|0| Added|
|Input.Input Example Icon Clearable Shorthand.chromium.png|0| Added|
|Input.Input Example Icon Position.chromium.png|0| Added|
|Input.Input Example Icon.chromium.png|0| Added|
|Input.Input Example Inline.chromium.png|0| Added|
|Input.Input Example Input Slot.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams Dark.Input- Inside Label with Value.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams Dark.Input- Labels.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams High Contrast.Input- Inside Label with Value.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams High Contrast.Input- Labels.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams.Input- Inside Label with Value.chromium.png|0| Added|
|Input.Input Example Label Shorthand Teams.Input- Labels.chromium.png|0| Added|
|Input.Input Example Label Shorthand.Input- Inside Label with Value.chromium.png|0| Added|
|Input.Input Example Label Shorthand.Input- Labels.chromium.png|0| Added|
|Input.Input Example Rtl.chromium.png|0| Added|
|Input.Input Example Targeting.chromium.png|0| Added|
|Input.Input Example Teams Dark.chromium.png|0| Added|
|Input.Input Example Teams High Contrast.chromium.png|0| Added|
|Input.Input Example Teams.chromium.png|0| Added|
|Input.Input Example Wrapper Slot.chromium.png|0| Added|
|Input.Input Example.chromium.png|0| Added|
|ItemLayout.Item Layout Example Content Media Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Content.chromium.png|0| Added|
|ItemLayout.Item Layout Example End Media Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Header Content Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Header Media Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Header.chromium.png|0| Added|
|ItemLayout.Item Layout Example Media Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Rtl Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Selection Shorthand.chromium.png|0| Added|
|ItemLayout.Item Layout Example Shorthand.chromium.png|0| Added|
|Label.Label Example Circular Shorthand.chromium.png|0| Added|
|Label.Label Example Color.chromium.png|0| Added|
|Label.Label Example Content Customization Shorthand.chromium.png|0| Added|
|Label.Label Example Icon Position Shorthand.chromium.png|0| Added|
|Label.Label Example Icon Shorthand.chromium.png|0| Added|
|Label.Label Example Image Shorthand.chromium.png|0| Added|
|Label.Label Example Rtl.chromium.png|0| Added|
|Label.Label Example Shorthand.chromium.png|0| Added|
|Label.Label Example.chromium.png|0| Added|
|Layout.Layout Example Disappearing.chromium.png|0| Added|
|Layout.Layout Example Gap.chromium.png|0| Added|
|Layout.Layout Example Reducing.chromium.png|0| Added|
|Layout.Layout Example Vertical.chromium.png|0| Added|
|Layout.Layout Example.chromium.png|0| Added|
|List.List Example Media Shorthand.chromium.png|0| Added|
|List.List Example Media.chromium.png|0| Added|
|List.List Example Navigable Teams Dark.Focuses item.chromium.png|0| Added|
|List.List Example Navigable Teams Dark.chromium.png|0| Added|
|List.List Example Navigable Teams High Contrast.Focuses item.chromium.png|0| Added|
|List.List Example Navigable Teams High Contrast.chromium.png|0| Added|
|List.List Example Navigable Teams.Focuses item.chromium.png|0| Added|
|List.List Example Navigable Teams.chromium.png|0| Added|
|List.List Example Navigable.Focuses item.chromium.png|0| Added|
|List.List Example Navigable.chromium.png|0| Added|
|List.List Example Rtl.chromium.png|0| Added|
|List.List Example Selectable Teams Dark.Focuses item.chromium.png|0| Added|
|List.List Example Selectable Teams Dark.chromium.png|0| Added|
|List.List Example Selectable Teams High Contrast.Focuses item.chromium.png|0| Added|
|List.List Example Selectable Teams High Contrast.chromium.png|0| Added|
|List.List Example Selectable Teams.Focuses item.chromium.png|0| Added|
|List.List Example Selectable Teams.chromium.png|0| Added|
|List.List Example Selectable.Focuses item.chromium.png|0| Added|
|List.List Example Selectable.chromium.png|0| Added|
|List.List Example.chromium.png|0| Added|
|List.Selectable List Controlled Example.chromium.png|0| Added|
|Loader.Loader Example Delay.chromium.png|0| Added|
|Loader.Loader Example Inline.chromium.png|0| Added|
|Loader.Loader Example Label.chromium.png|0| Added|
|Loader.Loader Example Rtl.chromium.png|0| Added|
|Loader.Loader Example Secondary.chromium.png|0| Added|
|Loader.Loader Example Size.chromium.png|0| Added|
|Loader.Loader Example.chromium.png|0| Added|
|Menu.Menu Controlled Example Disabled.chromium.png|0| Added|
|Menu.Menu Example Divider Horizontal.chromium.png|0| Added|
|Menu.Menu Example Fluid.chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only Teams.chromium.png|0| Added|
|Menu.Menu Example Icon Only.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Icon Only.chromium.png|0| Added|
|Menu.Menu Example Kind.chromium.png|0| Added|
|Menu.Menu Example Pointing Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Pointing Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Pointing Teams.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing Teams.chromium.png|0| Added|
|Menu.Menu Example Pointing.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Pointing.chromium.png|0| Added|
|Menu.Menu Example Positioning Shorthand.chromium.png|0| Added|
|Menu.Menu Example Positioning Update Shorthand.chromium.png|0| Added|
|Menu.Menu Example Slot.chromium.png|0| Added|
|Menu.Menu Example Tab Shorthand.chromium.png|0| Added|
|Menu.Menu Example Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Teams.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Teams.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Teams.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Teams.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Clicks on the last item and opens submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Focuses on the first element in the submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Clicks on the last item and opens submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Focuses on the first element in the submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Clicks on the last item and opens submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Focuses on the first element in the submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand Teams.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Clicks on the last item and opens submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Focuses on the first element in the submenu.chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Toolbar Shorthand.chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Underlined Teams.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined Teams.chromium.png|0| Added|
|Menu.Menu Example Underlined.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Underlined.chromium.png|0| Added|
|Menu.Menu Example Vertical Pointing.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Pointing.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Pointing.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Pointing.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Pointing.chromium.png|0| Added|
|Menu.Menu Example Vertical Teams Dark.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams Dark.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams Dark.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams Dark.chromium.png|0| Added|
|Menu.Menu Example Vertical Teams High Contrast.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams High Contrast.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams High Contrast.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical Teams.chromium.png|0| Added|
|Menu.Menu Example Vertical.Clicks on 4th item (opens submenu).chromium.png|0| Added|
|Menu.Menu Example Vertical.Hovers 1st item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical.Hovers 4th item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example Vertical.chromium.png|0| Added|
|Menu.Menu Example With Icons.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example With Icons.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example With Icons.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example With Icons.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example With Icons.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams Dark.Click 1st item, open menu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams Dark.Hovers 2nd item, open submenu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams Dark.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.Click 1st item, open menu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.Hovers 2nd item, open submenu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams.Click 1st item, open menu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams.Hovers 2nd item, open submenu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover Teams.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover.Click 1st item, open menu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover.Hovers 2nd item, open submenu.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu Hover.chromium.png|0| Added|
|Menu.Menu Example With Sub Menu.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example With Sub Menu.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example With Sub Menu.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example With Sub Menu.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example With Sub Menu.chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams Dark.Hovers 1st item (show tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams Dark.Navigates to next item (shows tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams Dark.chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams High Contrast.Hovers 1st item (show tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams High Contrast.Navigates to next item (shows tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams High Contrast.chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams.Hovers 1st item (show tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams.Navigates to next item (shows tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip Teams.chromium.png|0| Added|
|Menu.Menu Example With Tooltip.Hovers 1st item (show tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip.Navigates to next item (shows tooltip).chromium.png|0| Added|
|Menu.Menu Example With Tooltip.chromium.png|0| Added|
|Menu.Menu Example.Clicks on 2nd item (active state styles).chromium.png|0| Added|
|Menu.Menu Example.Hovers 2nd item (hover state styles).chromium.png|0| Added|
|Menu.Menu Example.Navigates to next item (focus state styles).chromium.png|0| Added|
|Menu.Menu Example.Navigates to previous item (active and focus state styles).chromium.png|0| Added|
|Menu.Menu Example.chromium.png|0| Added|
|Menu.Menu Item Example Disabled.chromium.png|0| Added|
|Menu.Menu Playground.chromium.png|0| Added|
|Menu.Menu With Submenu Controlled Example.chromium.png|0| Added|
|MenuButton.Menu Button Example Rtl.RTL- Shows menuButton.chromium.png|0| Added|
|MenuButton.Menu Button Example Rtl.chromium.png|0| Added|
|MenuButton.Menu Button Open Example.chromium.png|0| Added|
|Pill.Pill Actionable Example.chromium.png|0| Added|
|Pill.Pill Appearance Example.chromium.png|0| Added|
|Pill.Pill Example Disabled.chromium.png|0| Added|
|Pill.Pill Example.chromium.png|0| Added|
|Pill.Pill Group Example.chromium.png|0| Added|
|Pill.Pill Icon Example.chromium.png|0| Added|
|Pill.Pill Image Example.chromium.png|0| Added|
|Pill.Pill Rectangular Example.chromium.png|0| Added|
|Pill.Pill Selectable Example.chromium.png|0| Added|
|Pill.Pill Sizes Example.chromium.png|0| Added|
|Popup.Popper Example Positioning.chromium.png|0| Added|
|Popup.Popper Example Visibility Modifiers.Opened a popup on second message.chromium.png|0| Added|
|Popup.Popper Example Visibility Modifiers.chromium.png|0| Added|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-escaped]  when the popper escapes the reference element's boundary.chromium.png|0| Added|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-is-intersecting]  when the popover intersects boundaries.chromium.png|0| Added|
|Popup.Popper Example Visibility Modifiers.has  [data-popper-reference-hidden]  when the reference is hidden.chromium.png|0| Added|
|Popup.Popup Close Button Example.chromium.png|0| Added|
|Popup.Popup Controlled Example Teams Dark.chromium.png|0| Added|
|Popup.Popup Controlled Example Teams High Contrast.chromium.png|0| Added|
|Popup.Popup Controlled Example Teams.chromium.png|0| Added|
|Popup.Popup Controlled Example.chromium.png|0| Added|
|Popup.Popup Custom Target Example.chromium.png|0| Added|
|Popup.Popup Example Async.chromium.png|0| Added|
|Popup.Popup Example Auto Size.chromium.png|0| Added|
|Popup.Popup Example Container Transformed.chromium.png|0| Added|
|Popup.Popup Example Inline.chromium.png|0| Added|
|Popup.Popup Example Nested.chromium.png|0| Added|
|Popup.Popup Example Offset.Opens a popup.chromium.png|0| Added|
|Popup.Popup Example Offset.chromium.png|0| Added|
|Popup.Popup Example Pointer Margin.chromium.png|0| Added|
|Popup.Popup Example Pointer Offset.chromium.png|0| Added|
|Popup.Popup Example Pointing.chromium.png|0| Added|
|Popup.Popup Example Position.chromium.png|0| Added|
|Popup.Popup Example Rtl.RTL- Shows popup.chromium.png|0| Added|
|Popup.Popup Example Rtl.chromium.png|0| Added|
|Popup.Popup Example Teams Dark.Shows popup.chromium.png|0| Added|
|Popup.Popup Example Teams Dark.chromium.png|0| Added|
|Popup.Popup Example Teams High Contrast.Shows popup.chromium.png|0| Added|
|Popup.Popup Example Teams High Contrast.chromium.png|0| Added|
|Popup.Popup Example Teams.Shows popup.chromium.png|0| Added|
|Popup.Popup Example Teams.chromium.png|0| Added|
|Popup.Popup Example.Shows popup.chromium.png|0| Added|
|Popup.Popup Example.chromium.png|0| Added|
|Popup.Popup Focus Trap Example.chromium.png|0| Added|
|Popup.Popup Scroll Example.Click on trigger button for controlled popup with manual focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.Click on trigger button for inline controlled popup with manual focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.Click on trigger button for popup with trap focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.Scroll to trigger button for controlled popup with manual focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.Scroll to trigger button for inline controlled popup with manual focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.Scroll to trigger button for popup with trap focus.chromium.png|0| Added|
|Popup.Popup Scroll Example.chromium.png|0| Added|
|Portal.Portal Example Open.chromium.png|0| Added|
|Provider.Provider Example Animation.chromium.png|0| Added|
|Provider.Provider Example Focus Border Shorthand.Show overridden border focused styles.chromium.png|0| Added|
|Provider.Provider Example Focus Border Shorthand.chromium.png|0| Added|
|Provider.Provider Example Scrollbar.chromium.png|0| Added|
|Provider.Provider Example Shorthand.chromium.png|0| Added|
|Provider.Provider Example Target Frame.chromium.png|0| Added|
|Provider.Provider Example Target.chromium.png|0| Added|
|Provider.Provider Rtl Example.chromium.png|0| Added|
|RadioGroup.Radio Group Color Picker Example.chromium.png|0| Added|
|RadioGroup.Radio Group Custom Example.chromium.png|0| Added|
|RadioGroup.Radio Group Example Rtl.chromium.png|0| Added|
|RadioGroup.Radio Group Item Example Checked Shorthand.chromium.png|0| Added|
|RadioGroup.Radio Group Item Example Disabled Shorthand.chromium.png|0| Added|
|RadioGroup.Radio Group Item Example.chromium.png|0| Added|
|RadioGroup.Radio Group Vertical Example Teams Dark.chromium.png|0| Added|
|RadioGroup.Radio Group Vertical Example Teams High Contrast.chromium.png|0| Added|
|RadioGroup.Radio Group Vertical Example Teams.chromium.png|0| Added|
|RadioGroup.Radio Group Vertical Example.chromium.png|0| Added|
|Reaction.Reaction Example.chromium.png|0| Added|
|Reaction.Reaction Group Example Rtl.chromium.png|0| Added|
|Reaction.Reaction Group Example.chromium.png|0| Added|
|Segment.Segment Example Color.chromium.png|0| Added|
|Segment.Segment Example Disabled Shorthand.chromium.png|0| Added|
|Segment.Segment Example Disabled.chromium.png|0| Added|
|Segment.Segment Example Inverted Shorthand.chromium.png|0| Added|
|Segment.Segment Example Rtl.chromium.png|0| Added|
|Segment.Segment Example Shorthand.chromium.png|0| Added|
|Segment.Segment Example.chromium.png|0| Added|
|Skeleton.Skeleton Example Animations.chromium.png|0| Added|
|Skeleton.Skeleton Example Card.chromium.png|0| Added|
|Skeleton.Skeleton Example Components.chromium.png|0| Added|
|Skeleton.Skeleton Example Default.chromium.png|0| Added|
|Skeleton.Skeleton Example List.chromium.png|0| Added|
|Slider.Slider Example Action Shorthand.chromium.png|0| Added|
|Slider.Slider Example Disabled Shorthand.chromium.png|0| Added|
|Slider.Slider Example Fluid Shorthand.chromium.png|0| Added|
|Slider.Slider Example Form Shorthand.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams Dark.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams Dark.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams High Contrast.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams High Contrast.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Rtl Teams.chromium.png|0| Added|
|Slider.Slider Example Rtl.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Rtl.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams Dark.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams Dark.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams High Contrast.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Shorthand Teams.chromium.png|0| Added|
|Slider.Slider Example Shorthand.Navigates to the right with the up arrow key.chromium.png|0| Added|
|Slider.Slider Example Shorthand.chromium.png|0| Added|
|Slider.Slider Example Vertical Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Example Disabled Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Example Flat.chromium.png|0| Added|
|SplitButton.Split Button Example Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Example Small Container Teams Dark.chromium.png|0| Added|
|SplitButton.Split Button Example Small Container Teams High Contrast.chromium.png|0| Added|
|SplitButton.Split Button Example Small Container Teams.chromium.png|0| Added|
|SplitButton.Split Button Example Small Container.chromium.png|0| Added|
|SplitButton.Split Button Example Small Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Example Toggle Button Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Icon And Content Example Shorthand.chromium.png|0| Added|
|SplitButton.Split Button Main Option Change Example.chromium.png|0| Added|
|SplitButton.Split Button Positioning Example Shorthand.Open menu.chromium.png|0| Added|
|SplitButton.Split Button Positioning Example Shorthand.chromium.png|0| Added|
|Status.Status Color Example Shorthand.chromium.png|0| Added|
|Status.Status Custom Example Shorthand.chromium.png|0| Added|
|Status.Status Example Shorthand.chromium.png|0| Added|
|Status.Status Icon Example Shorthand.chromium.png|0| Added|
|Status.Status Size Example Shorthand.chromium.png|0| Added|
|Status.Status Type Example Shorthand Teams Dark.chromium.png|0| Added|
|Status.Status Type Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Status.Status Type Example Shorthand Teams.chromium.png|0| Added|
|Status.Status Type Example Shorthand.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Circular Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Circular Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Circular Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Circular.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Color Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Color Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Color Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Color.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Disabled Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Disabled Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Disabled Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Disabled.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Rotate Rtl.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Rotate.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Rtl.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Size Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Size Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Size Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Size.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Space Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Space Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Space Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Space.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Example Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Example.chromium.png|0| Added|
|SvgIcon.Svg Icon Set Example Shorthand Teams Dark.chromium.png|0| Added|
|SvgIcon.Svg Icon Set Example Shorthand Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svg Icon Set Example Shorthand Teams.chromium.png|0| Added|
|SvgIcon.Svg Icon Set Example Shorthand.chromium.png|0| Added|
|SvgIcon.Svg Playground.chromium.png|0| Added|
|SvgIcon.Svgcon Example Bordered Teams Dark.chromium.png|0| Added|
|SvgIcon.Svgcon Example Bordered Teams High Contrast.chromium.png|0| Added|
|SvgIcon.Svgcon Example Bordered Teams.chromium.png|0| Added|
|SvgIcon.Svgcon Example Bordered.chromium.png|0| Added|
|Table.Static Table Compact.chromium.png|0| Added|
|Table.Static Table Teams Dark.chromium.png|0| Added|
|Table.Static Table Teams High Contrast.chromium.png|0| Added|
|Table.Static Table Teams.chromium.png|0| Added|
|Table.Static Table.chromium.png|0| Added|
|Table.Table Example Static Shorthand.chromium.png|0| Added|
|Table.Table Example Static.chromium.png|0| Added|
|Text.Header Example Align.chromium.png|0| Added|
|Text.Text Example At Mention Shorthand.chromium.png|0| Added|
|Text.Text Example At Mention.chromium.png|0| Added|
|Text.Text Example Color.chromium.png|0| Added|
|Text.Text Example Disabled Shorthand.chromium.png|0| Added|
|Text.Text Example Disabled.chromium.png|0| Added|
|Text.Text Example Error Shorthand.chromium.png|0| Added|
|Text.Text Example Error.chromium.png|0| Added|
|Text.Text Example Important Shorthand.chromium.png|0| Added|
|Text.Text Example Important.chromium.png|0| Added|
|Text.Text Example Rtl.chromium.png|0| Added|
|Text.Text Example Success Shorthand.chromium.png|0| Added|
|Text.Text Example Success.chromium.png|0| Added|
|Text.Text Example Temporary Shorthand.chromium.png|0| Added|
|Text.Text Example Temporary.chromium.png|0| Added|
|Text.Text Example Timestamp Shorthand.chromium.png|0| Added|
|Text.Text Example Timestamp.chromium.png|0| Added|
|Text.Text Example Truncated Shorthand.chromium.png|0| Added|
|Text.Text Example Truncated.chromium.png|0| Added|
|Text.Text Sizes Example Shorthand.chromium.png|0| Added|
|Text.Text Sizes Example.chromium.png|0| Added|
|Text.Text Weights Example Shorthand.chromium.png|0| Added|
|Text.Text Weights Example.chromium.png|0| Added|
|TextArea.Text Area Disabled Example.chromium.png|0| Added|
|TextArea.Text Area Example Fluid.chromium.png|0| Added|
|TextArea.Text Area Example Height.chromium.png|0| Added|
|TextArea.Text Area Example Inverted Teams Dark.chromium.png|0| Added|
|TextArea.Text Area Example Inverted Teams High Contrast.chromium.png|0| Added|
|TextArea.Text Area Example Inverted Teams.chromium.png|0| Added|
|TextArea.Text Area Example Inverted.chromium.png|0| Added|
|TextArea.Text Area Example Max Length.chromium.png|0| Added|
|TextArea.Text Area Example Resize.chromium.png|0| Added|
|TextArea.Text Area Example Teams Dark.Can be focused.chromium.png|0| Added|
|TextArea.Text Area Example Teams Dark.chromium.png|0| Added|
|TextArea.Text Area Example Teams High Contrast.Can be focused.chromium.png|0| Added|
|TextArea.Text Area Example Teams High Contrast.chromium.png|0| Added|
|TextArea.Text Area Example Teams.Can be focused.chromium.png|0| Added|
|TextArea.Text Area Example Teams.chromium.png|0| Added|
|TextArea.Text Area Example.Can be focused.chromium.png|0| Added|
|TextArea.Text Area Example.chromium.png|0| Added|
|TextArea.Text Area Value Example.chromium.png|0| Added|
|Toolbar.Toolbar Example Action Popup In Menu Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Action Popup In Menu Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Action Popup In Menu Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Action Popup In Menu.chromium.png|0| Added|
|Toolbar.Toolbar Example Custom Content Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Editor Shorthand Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Editor Shorthand Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Editor Shorthand Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Editor Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu Item Toggle.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu Radio Group.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu With Submenu Shorthand Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Menu With Submenu Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Overflow Positioning Shorthand Rtl.RTL- Overflow item is properly positioned.chromium.png|0| Added|
|Toolbar.Toolbar Example Overflow Positioning Shorthand Rtl.chromium.png|0| Added|
|Toolbar.Toolbar Example Overflow Positioning Shorthand.Overflow item is properly positioned.chromium.png|0| Added|
|Toolbar.Toolbar Example Overflow Positioning Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Overflow.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup In Menu Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup In Menu Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup In Menu Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup In Menu.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup Shorthand Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup Shorthand Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup Shorthand Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Popup Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Radio Group Shorthand Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Radio Group Shorthand Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Radio Group Shorthand Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Radio Group Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Shorthand Teams Dark.chromium.png|0| Added|
|Toolbar.Toolbar Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Toolbar.Toolbar Example Shorthand Teams.chromium.png|0| Added|
|Toolbar.Toolbar Example Shorthand.chromium.png|0| Added|
|Toolbar.Toolbar Example Variables.chromium.png|0| Added|
|Toolbar.Toolbar Example.chromium.png|0| Added|
|Tooltip.Tooltip Example Dialog Content Shorthand.Shows tooltip in a dialog.chromium.png|0| Added|
|Tooltip.Tooltip Example Dialog Content Shorthand.chromium.png|0| Added|
|Tooltip.Tooltip Example Disabled Trigger.chromium.png|0| Added|
|Tooltip.Tooltip Example Dismiss On Content Mouse Enter.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointer Margin.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams Dark.Has outline on keyboard.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams Dark.No outline after click.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams Dark.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams High Contrast.Has outline on keyboard.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams High Contrast.No outline after click.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams High Contrast.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams.Has outline on keyboard.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams.No outline after click.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing Teams.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing.Has outline on keyboard.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing.No outline after click.chromium.png|0| Added|
|Tooltip.Tooltip Example Pointing.chromium.png|0| Added|
|Tooltip.Tooltip Example Position.chromium.png|0| Added|
|Tooltip.Tooltip Example Rtl.RTL- Shows tooltip.chromium.png|0| Added|
|Tooltip.Tooltip Example Rtl.chromium.png|0| Added|
|Tooltip.Tooltip Example Target.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams Dark.Shows tooltip.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams Dark.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams High Contrast.Shows tooltip.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams High Contrast.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams.Shows tooltip.chromium.png|0| Added|
|Tooltip.Tooltip Example Teams.chromium.png|0| Added|
|Tooltip.Tooltip Example.Shows tooltip.chromium.png|0| Added|
|Tooltip.Tooltip Example.chromium.png|0| Added|
|Tooltip.Tooltip Open Example.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.Focus on click leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.Focus on click subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.Focus on keyboard leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.Focus on keyboard subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams Dark.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on click leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on click subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on keyboard leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams High Contrast.Focus on keyboard subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams High Contrast.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams.Focus on click leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams.Focus on click subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams.Focus on keyboard leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams.Focus on keyboard subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand Teams.chromium.png|0| Added|
|Tree.Tree Example Shorthand.Focus on click leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand.Focus on click subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand.Focus on keyboard leaf.chromium.png|0| Added|
|Tree.Tree Example Shorthand.Focus on keyboard subtree.chromium.png|0| Added|
|Tree.Tree Example Shorthand.chromium.png|0| Added|
|Tree.Tree Exclusive Example.chromium.png|0| Added|
|Tree.Tree Initially Open Example Shorthand.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams Dark.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams Dark.default selected states.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams Dark.first expanded, not custom checkbox visible.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams High Contrast.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams High Contrast.default selected states.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams High Contrast.first expanded, not custom checkbox visible.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams.default selected states.chromium.png|0| Added|
|Tree.Tree Multiselect Example Teams.first expanded, not custom checkbox visible.chromium.png|0| Added|
|Tree.Tree Multiselect Example.chromium.png|0| Added|
|Tree.Tree Multiselect Example.default selected states.chromium.png|0| Added|
|Tree.Tree Multiselect Example.first expanded, not custom checkbox visible.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams Dark.Exapanded.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams Dark.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams High Contrast.Exapanded.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams High Contrast.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams.Exapanded.chromium.png|0| Added|
|Tree.Tree Title Customization Example Teams.chromium.png|0| Added|
|Tree.Tree Title Customization Example.Exapanded.chromium.png|0| Added|
|Tree.Tree Title Customization Example.chromium.png|0| Added|
|Video.Video Example.chromium.png|0| Added|

</details>


<div id="vrtCommentE2E Tests-v0"/>